src/genie/libs/parser/iosxe/show_snmp.py

```diff
@@ -848,14 +848,15 @@ def cli(self, interface=None, output=None):
         interface_dict = {}
 
         #GigabitEthernet3/0/28: Ifindex = 36
-        p1 =  re.compile(r'(?P<interface_index>\S+) +Ifindex = +(?P<ifIndex>\d+)$')
+        #unrouted VLAN 1003: Ifindex = 7
+        p1 =  re.compile(r'^(unrouted\s+)?(?P<interface_index>(VLAN\s)?\S+) +Ifindex = +(?P<ifIndex>\d+)$')
```

Comment:
After speaking with the team, we decided to keep the unrouted part in as well. This also removes the need for .replace()

```suggestion
        p1 =  re.compile(r'^(?P<interface_index>(unrouted\sVLAN\s)?\S+) +Ifindex = +(?P<ifIndex>\d+)$')
```